nautobot_floor_plan/choices.py

```diff
@@ -19,8 +19,32 @@ class RackOrientationChoices(ChoiceSet):
     )
 
 
+class CustomAxisLabelsChoices(ChoiceSet):
+    """Choices for custom axis label types."""
+
+    ROMAN = "roman"
+    GREEK = "greek"
+    BINARY = "binary"
+    HEX = "hex"
+    NUMALPHA = "numalpha"
+    LETTERS = "letters"
+    ALPHANUMERIC = "alphanumeric"
+    NUMBERS = "numbers"
+
+    CHOICES = (
+        (ROMAN, "Roman (e.g., I, II, III)"),
+        (GREEK, "Greek (e.g., α, β, γ)"),
+        (BINARY, "Binary (e.g., 1, 10, 11)"),
+        (HEX, "Hexadecimal (e.g., 1, A, F)"),
+        (NUMALPHA, "numalpha (e.g., 02A)"),
+        (LETTERS, "Letters (e.g., A, B, C)"),
+        (ALPHANUMERIC, "Alphanumeric (e.g., A01, B02)"),
+        (NUMBERS, "Numbers (e.g. 1, 2, 3)"),
+    )
+
+
 class AxisLabelsChoices(ChoiceSet):
-    """Choices for grid numbering style."""
+    """Choices for axis labels."""
```

Comment:
A bit non-obvious to me why we have both `CustomAxisLabelsChoices` and `AxisLabelsChoices`, is this primarily for backwards compatibility of some sort? Might be helpful to add some comments or docstring content distinguishing between the purpose of the two.